tests/store/artifact/test_cli.py

```diff
@@ -61,6 +54,19 @@ def test_get_artifact_repository(artifact_uri):
         ("s3://path/to", ("s3://path/", "to")),
         ("s3://path/to/dir", ("s3://path/to", "dir")),
     ]
+    if platform.system() != "Windows":
+        # Not applicable on Windows
```

Comment:
The code path handling all of these cases below had to be disabled on Windows because a Windows file path could be `C:/path` and we don't want `C:` to be interpreted as the scheme part of the URI.